docs/source/guide/persistent_storage.md

```diff
@@ -448,6 +501,23 @@ az storage container create --name <YOUR_CONTAINER_NAME> \
           --account-key "<YOUR_STORAGE_KEY>"
 ```
 
+### Optional: Configure CORS for the Azure bucket
+
+> In the case if you're going to use direct file upload feature and store media files like audio, video, csv you should complete this step.
+
+Set up cross-origin resource sharing (CORS) access to your bucket. See [Configuring cross-origin resource sharing (CORS)](https://docs.microsoft.com/en-us/rest/api/storageservices/cross-origin-resource-sharing--cors--support-for-the-azure-storage-services#enabling-cors-for-azure-storage) in the Azure User Guide. Use or modify the following example:
```

Comment:
You don't have to call out cross-origin resource sharing (CORS), but you can use CORS here. The reason because you abbreviate it once at the beginning of the file and use the short terms throughout the file.